core/src/main/java/org/mule/routing/EventGroup.java

```diff
@@ -491,22 +496,19 @@ private void addAndOverrideSessionProperties(MuleSession session, MuleEvent even
         }
     }
 
-    private ObjectStoreManager getObjectStoreManager()
-    {
-        if (objectStoreManager == null)
-        {
-            objectStoreManager = muleContext.getRegistry().get(
-                    MuleProperties.OBJECT_STORE_MANAGER);
-        }
-        return objectStoreManager;
-    }
-
     public void initAfterDeserialisation(MuleContext context) throws MuleException
     {
         this.muleContext = context;
+    }
 
-        String storeKey = storePrefix + ".eventGroup." + groupId;
-        this.events = getObjectStoreManager().getObjectStore(storeKey, true);
+    private PartitionableObjectStore<MuleEvent> getEvents() throws ObjectStoreException
+    {
+        if (events == null)
+        {
+            events = muleContext.getRegistry().get(EVENTS_STORE_REGISTRY_KEY_PREFIX + storePrefix + ".eventGroup");
```

Comment:
I'm not sure that the objectStore will be always available on the registry. Can you check with LG?